Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -ldflags="-w -s" -o
 FROM alpine:3.13
 RUN mkdir /app && cd /app
 WORKDIR /app
-COPY server.config.yaml default.config.yaml
+COPY server.config.yaml server.config.yaml
```

Comment:
Wait wut? :P

Reply:
```COPY [src] [dst]```

The config file in the Docker image is just "the" config file (in this case server config), there's no such thing as "default" config (within the Docker image). Just "the" config.

It _can_ be the default config, in that case you'd name the source file `default.config.yaml`, not the destination.